src/View/clientForm/clientFormView.java

```diff
@@ -0,0 +1,274 @@
+package View.clientForm;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import javax.swing.*;
+
+public class clientFormView extends JPanel implements ActionListener {
+    private JPanel panelFormMain;
+    private JLabel clientDetails;
+    private JLabel clientName;
+    private JLabel clientEmail;
+    private JLabel clientPhone;
+    private JLabel clientContact;
+    private JLabel clientAddress;
+    private JLabel clientCity;
+    private JLabel clientState;
+    private JLabel clientZIP;
+    private JLabel clientType;
+    private JTextField nameText;
+    private JTextField emailText;
+    private JTextField phoneText;
+    private JTextField contactText;
+    private JTextField addressText;
+    private JTextField cityText;
+    private JTextField stateText;
+    private JTextField zipText;
+    private JTextField typeText;
+    private JButton clientButton;
+
+    public clientFormView(String action) {
+
+        clientDetails = new JLabel("    Client Details");
+        clientName = new JLabel("Client Name: ");
+        clientEmail = new JLabel("Email : ");
+        clientPhone = new JLabel("Phone Number: ");
+        clientContact = new JLabel("Preferred Contact: ");
+        clientAddress = new JLabel("Address: ");
+        clientCity = new JLabel("City: ");
+        clientState = new JLabel("State: ");
+        clientZIP = new JLabel("ZIP: ");
+        clientType = new JLabel("Type: ");
+
+        nameText = new JTextField(20);
```

Comment:
if the int is the max length of a text field we should do 45 for each of these.

Reply:
The int is the width of the text field.